lib/web/apiserver.go

```diff
@@ -5166,11 +5168,53 @@ func (h *Handler) authExportPublic(w http.ResponseWriter, r *http.Request, p htt
 		return
```

Comment:
Might be overkill, but `http.Error` is going to send a plain text response. Should we also send errors in JSON if the client requested JSON?

Reply:
Looks like we were already using http.Error on line 5155 too.